templates/node/node.py

```diff
@@ -37,17 +38,24 @@ def validate(self):
         if not self.data['sshKey']:
             raise ValueError('sshKey service name is required')
 
-    def get_name(self):
-        """ Return VM name """
-
+    def _execute_task(self, proxy, action, args={}):
```

Comment:
A better way to do this is move execute_task to the 0-robot TemplateBase class.

option 1: blocking alternative for schedule_action
```python
class TemplateBase(object):
...
    def execute_action(self, action, **kwargs):
        pass

# usage:
result = myservice.execute_action("install", reinstall=True)
```
option 2: implement __get_attribute__ and provide a seemless blocking interface
```python
class TemplateBase(object):
...
    def __get_attribute__(self, action):
        def action(**kwargs):
            pass

# usage:
result = myservice.install(reinstall=True)
```